libs/ledger-live-common/src/hooks/useAvailableLanguagesForDevice.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from "react";
```

Comment:
Maybe put this hook under a new folder `hw/hooks` (or if you think it's more related to manager, `manager/hooks`) ?

So it's closer to the code the hook is using. And i'm actually adding a hook for genuine check in `hw/hooks` right now